server/policies/group.ts

```diff
@@ -1,37 +1,27 @@
 import { Group, User, Team } from "@server/models";
-import { AdminRequiredError } from "../errors";
 import { allow } from "./cancan";
+import { and, isTeamModel } from "./utils";
 
-allow(User, "createGroup", Team, (actor, team) => {
-  if (!team || actor.isGuest || actor.isViewer || actor.teamId !== team.id) {
-    return false;
-  }
-  if (actor.isAdmin) {
-    return true;
-  }
+allow(User, "createGroup", Team, (actor, team) =>
+  and(
+    //
```

Comment:
These comments are just to force vertical formatting otherwise prettier will place on a single line which is undesirable here.